docs/content/how-to/altering-tables.md

```diff
@@ -195,6 +231,16 @@ The following SQL sets column `coupon_info` to be nullable.
 
 {{< tabs "change-nullability-example" >}}
 
+{{< tab "Flink" >}}
+
+```sql
+CREATE TABLE T (a STRING PRIMARY KEY NOT ENFORCED, b FLOAT NOT NULL);
+ALTER TABLE T MODIFY b FLOAT;
+```
```

Comment:
1. The following SQL sets column `coupon_info` to be nullable.
2. modify nullability, not type.

I think changing column type is missing, please also add a new part for `changing column type` for both spark and flink.

Reply:
> * The following SQL sets column `coupon_info` to be nullable.
> * modify nullability, not type.

I don't seem to have found any relevant examples to modify nullability in flink [doc](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/alter/#modify)

Using an `alter table` syntax without `not null` to set a column to be nullable

Reply:
I asked flink community and the answer is 'alter table modify column_name data_type not null'. Can you have a try?

Reply:
> I asked flink community and the answer is `alter table modify column_name data_type not null`. Can you have a try?

I have add a test in the `SchemaChangeITCase`, change the column to not null will throw an exception.

```
      assertThatThrownBy(() -> sql("ALTER TABLE T MODIFY c STRING NOT NULL"))
                .hasMessageContaining(
                        "Could not execute ALTER TABLE PAIMON.default.T\n"
                                + "  MODIFY `c` STRING NOT NULL");

```

Reply:
Ok, seems flink only support change `not null` to `nullable`.
I have some thought about this. If the column is nullable before, we can insert null value. Then if we change it to not null, the previous null value is conflict to the `not null` definition. So I think flink's behavior is reasonable.